doc/asciidoc/similarity-pearson.adoc

```diff
@@ -66,6 +66,47 @@ include::scripts/similarity-pearson.cypher[tag=function]
 include::scripts/similarity-pearson.cypher[tag=create-sample-graph]
 ----
 
+.The following will return the Pearson similarity of Arya and Karin:
+[source, cypher]
+----
+include::scripts/similarity-pearson.cypher[tag=function-cypher]
+----
+
+// tag::function-cypher[]
+.Results
+[opts="header"]
+|===
+| `from`       | `to`         | `similarity`
+|"Arya" |	"Karin"	 | 0.8194651785206903
+
+|===
+// end::function-cypher[]
+
+
+In this example we pass in `vectorType: "maps"` as an extra parameter, as well as using the `algo.similarity.asVector` function to construct a vector of maps containing each movie and the corresponding rating.
+We do this because the Pearson similarity algorithm needs to compute the average of *all* the movies that a user has reviewed, not just the ones that they have in common with the user we're comparing them to.
```

Comment:
Algo name should be capitalised as per the header: **Pearson Similarity algorithm**